controllers/nginx/pkg/cmd/controller/nginx.go

```diff
@@ -166,23 +169,29 @@ type NGINXController struct {
 	ports *config.ListenPorts
 
 	backendDefaults defaults.Backend
+
+	isShuttingDown bool
 }
 
 // Start start a new NGINX master process running in foreground.
 func (n *NGINXController) Start() {
 	glog.Info("starting NGINX process...")
 
-	done := make(chan error, 1)
 	cmd := exec.Command(n.binary, "-c", cfgPath)
-	n.start(cmd, done)
+	start(cmd, n.done)
```

Comment:
We need to put the Nginx process in a different group than the controller to avoid the `SIGTERM` signal to be forwarded.
```
cmd.SysProcAttr = &syscall.SysProcAttr{
  Setpgid: true,
  Pgid:    0,
}
```